components/Item/Card/Card.vue

```diff
@@ -237,13 +237,8 @@ export default Vue.extend({
       }
     },
     getImageType(): ImageType {
-      if (
-        this.shape === 'thumb-card' &&
-        !['CollectionFolder', 'Folder', 'UserView'].includes(
-          this.item?.Type || ''
-        )
-      ) {
-        return ImageType.Thumb;
+      if (this.shape === 'thumb-card' && this.item.Type === 'Movie') {
```

Comment:
How does that work for series, do they use the Primary?

Reply:
Do you mean for episodes?

If I remember correctly,
Episodes have primary image type that are thumb shaped

Reply:
Yeah episodes, my bad. And that's why it worked with my episodes in next up :p

Reply:
A thumb being these wide images? And posters the vertical ones IIRC?

Reply:
Yeah that's right